test/integration/build/aws.tf

```diff
@@ -47,6 +47,8 @@ variable "aws_cloud_watch_logs_role_policy_name" {}
 variable "aws_configuration_recorder_name" {}
 variable "aws_configuration_recorder_role" {}
 variable "aws_create_configuration_recorder" {}
+variable "aws_dynamodb_table_name" {}
+variable "aws_dynamodb_global_table_name" {}
```

Comment:
Is this global table name used anywhere?

Reply:
I'm going to looking at adding a `dynamodb_global_table` resource after this. Will remove in the mean time.

Reply:
👍 It would cause CI to fail atm as the var isn't being set